### ISSUE TITLE
WCAG 4.1.2 tests 6, 8, 9: false positives

### ISSUE THREAD

**Issue by jzuidweg** [veraPDF/veraPDF-wcag-algs#86]
I'm attaching a [file](https://github.com/veraPDF/veraPDF-wcag-algs/files/7374463/tabel-met-tekst-eronder.pdf) which fails **WCAG 4.1.2** tests **6** ('_Invalid Table row_'), **8** ('_Invalid Table cell_') and **9** ('_Missing Table element(s)_') but shouldn't give these errors.

The false positives appear to be caused by the line directly under the table.

In [this other file](https://github.com/veraPDF/veraPDF-wcag-algs/files/7374465/tabel-zonder-tekst-eronder.pdf), which is the s**ame document** but **without the line directly under the table**, WCAG 4.1.2 tests 6, 8 and 9 do **not** give errors!

**Comment by bdoubrov** [comment]
This error is indeed caused to the line that follows the table to be placed very close to the last table row. This will be fixed by taking table borders into account in table recognition logic.

**Comment by jzuidweg** [comment]
This issue has been tested and is solved. It can be closed.
